modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -65,7 +65,7 @@ class BaseConvolutionLayerImpl : public ConvolutionLayer
     {
         setParamsFrom(params);
         getConvolutionKernelParams(params, kernel.height, kernel.width, pad.height,
-                                   pad.width, stride.height, stride.width, dilation.height,
+                                   pad.width, pad.height, pad.width, stride.height, stride.width, dilation.height,
```

Comment:
I think it's better to use local variables `pad_t`, `pad_l`, `pad_b`, `pad_r` here and raise an exception if some of pairs is asymmetric because now convolutional layer does not support it. Otherwise assign to `pad.height` and `pad.width`.

Reply:
done